proposals/NNNN-task-executor-preference.md

```diff
@@ -0,0 +1,825 @@
+# Task Executor Preference
+
+* Proposal: SE-NNNN
+* Author: [Konrad 'ktoso' Malawski](https://github.com/ktoso), [John McCall](https://github.com/rjmccall), [Franz Busch](https://github.com/FranzBusch)
+* Review Manager: TBD
+* Status: Partially implemented [PR #68793](https://github.com/apple/swift/pull/68793)
+* Implementation: TBD
+* Review: TBD
+
```

Comment:
Please add a link to the pitch thread, https://forums.swift.org/t/pitch-task-executor-preference/68191

Reply:
```suggestion
* Discussion threads:
    * Pitch: https://forums.swift.org/t/pitch-task-executor-preference/68191
```